### ISSUE TITLE
When sharing review on facebook I am led to a blank screen

### ISSUE THREAD

**Issue by theharrisonmoore** [yalla-coop/earwig#981]
When sharing my review to facebook, once I have clicked "post" on facebook I am led to a blank screen.

https://drive.google.com/open?id=1gzKG3UoC8_2qmt6U1uw0UIf6zTLS7pTz

iphone SE, Safari

**Comment by thejoefriel** [comment]
Hey @theharrisonmoore I think this may be more of a FB issue than us, as at that point you're interacting with the Facebook website, not the earwig one. Can I just check, does it then post it on facebook or not?

**Comment by theharrisonmoore** [comment]
Hi Joe, it *does* post it to FB but then instead of showing a thank you
page as it does on Linkedin and Whatsapp, it just shows a blank screen,
which gives the user no indication of whether it's worked or not, and
doesn't tell them it's time for them to go back to the earwig app.

PS This is not to confused with the FB Messenger issue #983

Best,
Harrison Moore
Sparky & Founder
*earwig <http://earwigwork.com>*

ᐧ

**Comment by thejoefriel** [comment]
Hi @theharrisonmoore this is a Facebook issue I believe rather than earwig. When you click share, it takes you to that website (you can see it by the url). The fact that FB just kick you off once you share as opposed to Whatsapp etc showing you it being done is a FB thing, which we can't control. I've posted a video to show you here: 

https://www.loom.com/share/1a6ab2a171c14e88a8b4ff893909ef74

We can maybe look into some sort of detection when they've posted it so there's then a consistent flow of closing the window and taking them to "successfully shared" style page, but this may not be possible as these are third party websites so it depends what level of access we can get. 

Does that make sense? Let me know if you want to chat through on the phone.

**Comment by theharrisonmoore** [comment]
@thejoefriel no need to chat on the phone. I can see you make a good point.

Going through this flow on desktop doesn't give the same UX as on mobile because with desktop you can still see the original earwig app open in the background which you can simply click back to. On mobile, on the other hand, the user is just left on a blank screen with no indication of whether the FB message has been successfully sent, whether they should click Back or whether they should close the current window/tab. It's just not a nice UX but like you said, not much we can do about this right now.

Perhaps you can label this as backlog and use some of the pilot time to think of a solution?

**Comment by thejoefriel** [comment]
Yeah I agree, not quite sure why Facebook have done it that way - proof that even the biggest product companies still have plenty to work on! 

Sure thing, will change this to backlog and discuss with the team.

**Comment by theharrisonmoore** [comment]
Nice one!

Best,
Harrison Moore
Sparky & Founder
*earwig <http://earwigwork.com>*

ᐧ
